index.js

```diff
@@ -11,6 +11,11 @@ module.exports = (grunt, options = {}) => {
 
 	let cwd = process.cwd();
 	let config = options.config || pkgUp.sync();
+
+	if (config === null) {
+		grunt.fail.fatal('package.json not found.\nPlease make sure to create a package.json and install all dependecies before run a grunt task.');
```

Comment:
You have some typos.

```suggestion
		grunt.fail.fatal('package.json not found.\nPlease make sure to create a package.json and install all dependencies before you run Grunt.');
```